src/main/java/staffconnect/model/person/PersonHasModulePredicate.java

```diff
@@ -0,0 +1,44 @@
+package staffconnect.model.person;
+
+import java.util.function.Predicate;
+
+import staffconnect.commons.util.ToStringBuilder;
+
+/**
+ * Tests that a {@code Person}'s {@code Module} matches the given module to filter for.
+ */
+public class PersonHasModulePredicate implements Predicate<Person> {
+    private final Module module;
+
+    public PersonHasModulePredicate(Module module) {
+        this.module = module;
+    }
+
+    @Override
+    public boolean test(Person person) {
+        if (module == null) {
+            return true;
+        }
+        return person.getModule().value.equalsIgnoreCase(module.value);
```

Comment:
similar for here:`person.getModule().toString().equalsIgnoreCase(module.toString());`